website/content/docs/templates/hcl_templates/locals.mdx

```diff
@@ -1,35 +1,34 @@
 ---
-page_title: Local Values - HCL Configuration Language
+page_title: Local Variables - HCL Configuration Language
```

Comment:
I wonder if we should update this document to be consistent with the Terraform [local values documentation](https://developer.hashicorp.com/terraform/language/values/locals). What do you think?

Reply:
@nywilken Oh, so you'd rather see me go the other direction, away from "variables" towards "values" for locals? Is Terraform consistent in the usage of "values"?

I don't really see a problem with doing that. Might lead to less confusion ... put more distance between the concepts so folks realize they're quite different in behavior.